types/react-mdl/index.d.ts

```diff
@@ -418,7 +418,7 @@ declare namespace __ReactMDL {
         hideSpacer?: boolean;
     }
     interface HeaderRowProps extends __MDLOtherProps {
-        title?: string;
+        title?: any; // string | JSX.Element
```

Comment:
I know the documentation says any, but if you know it's `string | JSX.Element`, why not set it as that? Or if it really can be `any`, then I would remove the comment.

Reply:
I fixed it according to the code `HeaderProps` and the [document](https://react-mdl.github.io/react-mdl/components/layout/).

> ```typescript
> interface HeaderProps extends __MDLOtherProps {
>     title?: any; // string | JSX.Element
>     scroll?: boolean;
>     seamed?: boolean;
>     transparent?: boolean;
>     waterfall?: boolean;
>     hideTop?: boolean;
>     hideSpacer?: boolean;
> }
> ```

> Element | Prop | Type | Effect | Remarks
> ------- | ---- | ---- | ------ | -------
> Header | title | Any | Set the layout title | Optional
> HeaderRow | title | Any | Set the layout title | Optional

But I think `ReactNode` may be better now. Is this correct ?
If it does not matter, I'll fix `HeaderProps` too.

Reply:
Sorry. I try to fix it, `HeaderProps` and `HeaderRowProps` inherit `React.HTMLAttributes`, `title` conflicts.

```typescript
interface HTMLAttributes<T> extends DOMAttributes<T> {
    // ...
    title?: string;
    // ...
}
```

What should I do？

Reply:
That makes sense. I think it is fine to use `any` in that case.